charts/hocs-info-service/values.yaml

```diff
@@ -1,6 +1,14 @@
 ---
 nameOverride: hocs-info-service
 
+selector:
+  ingress:
```

Comment:
Standard lines will need this to support ingress?

Reply:
that's either got to be the FE or the MUI hasn't it?

Reply:
Both I believe - MUI to add and view, FE to view.